<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRequirement</class>
 <widget class="QDialog" name="EditRequirement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="6">
    <widget class="QPushButton" name="editDom1Phen">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="12" column="6">
    <widget class="QPushButton" name="editDom2Phen">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="7">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="7">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="7">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Shared Phenomena:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="domainOneLabel">
     <property name="text">
      <string>Domain 1:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="6">
    <widget class="QTextEdit" name="descriptionTextEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="domainOne">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QRadioButton" name="indicativeRadio">
     <property name="text">
      <string>Indicative Property</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="Domain1SharedLabel">
     <property name="text">
      <string>Domain 1:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="Domain2SharedLabel">
     <property name="text">
      <string>Domain 2:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="5">
    <widget class="QListView" name="domain1SharedListView"/>
   </item>
   <item row="12" column="1" colspan="5">
    <widget class="QListView" name="domain2SharedListView"/>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="requirementRadio">
     <property name="text">
      <string>Requirement</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QComboBox" name="domainTwo">
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="domainTwoLabel">
     <property name="text">
      <string>Domain 2:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
